02-oop-inheritance/script.js

```diff
@@ -0,0 +1,62 @@
+function EventEmmiter(){
+	var eventHandlers = [event_play, event_pause, event_resume];
```

Comment:
you shouldn't add these specific event handler objects here, this should be initialized as an empty list. The addition of new event handler objects are responsibility of instances that extends EventEmitter.. like Movie